.claude/commands/backport.md

```
@@ -0,0 +1,325 @@
+import { Dropdown, DropdownItem, DropdownToggle } from "@patternfly/react-core";
+
+<Dropdown
+  isOpen={isOpen}
+  onSelect={() => setIsOpen(false)}
+  toggle={<DropdownToggle onToggle={setIsOpen}>{selected}</DropdownToggle>}
+  dropdownItems={[<DropdownItem key="1">Option 1</DropdownItem>]}
+/>;
+```
+
+Common v6 → v5 changes:
+| v6 (main) | v5 (release) | Notes |
```

markdownlint-cli2 (0.22.1)

[warning] Tables should be surrounded by blank lines

(MD058, blanks-around-tables)